cities/models.py

```diff
@@ -0,0 +1,11 @@
+from sqlalchemy import Column, Integer, String
+
+from database import Base
+
+
+class DbCity(Base):
+    __tablename__ = "city"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255), nullable=False)
+    additional_info = Column(String(511), nullable=False)
```

Comment:
The `additional_info` column is set to `nullable=False`, which means it is a required field. If this is not intended and you want to allow null values, consider setting `nullable=True`.